.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
```

Comment:
- [ ] Check this, another example of an inline comment.

Reply:
Hi mricanho,
So if I understand correctly, I need to make at least two inline comments to demonstrate if I understand how inline comment works but the comments not required to be related to the file. I mean if I comment in the linter.yml file it may not actually make sense but I can write anything just to show inline commenting